purge.py

```diff
@@ -20,3 +29,19 @@
 RESPONSE = requests.post(URL, data=PARAMS)
 
 print(RESPONSE.text)
+=======
+S = requests.Session()
+
+URL = "https://en.wikipedia.org/w/api.php"
+
+PARAMS = {
+    "action": "purge",
+    "generator": "allpages",
+    "gapnamespace": "0",
+    "gaplimit": "10",
+    "format": "json"
+}
+
+R = requests.post(url=URL, data=PARAMS)
+print(R.text)
+>>>>>>> Modified code samples for API:Purge
```

Comment:
@edyasikpo Looks like you forgot to resolve conflict before pushing changes for `purge.py`